esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2972,6 +2972,11 @@ bool CFileSprayEx::onFileList(IEspContext &context, IEspFileListRequest &req, IE
         if (!validateDropZonePath(nullptr, netaddr, sPath) && !validateConfigurationDirectory(nullptr, "log", nullptr, nullptr, sPath)) //The path should be the absolute path for the dropzone or log file.
             throw makeStringExceptionV(ECLWATCH_INVALID_INPUT, "Invalid file path %s", sPath.str());
 
+        const char* dropZoneName = req.getDropZoneName();
+        if (getDropZoneScopePermissions(context, dropZoneName, netaddr, sPath) < SecAccess_Read)
```

Comment:
I believe this only requires SecAccess_Access permission (get file attributes)?

Reply:
The logic here is the same as listing logical files (SecAccess_Read). The SecAccess_Read is also checked in line 2941.